### PR TITLE
add participants and superlunar/org bios

### PR DESCRIPTION
**Pull request by adamjonas**

### CODE REVIEW

**Comment by adamjonas** [comment]
Screen Shot 2022-06-12 at 4 51 42 PM
Screen Shot 2022-06-12 at 4 52 15 PM
Screen Shot 2022-06-12 at 4 53 53 PM

File: index.html [hunk 1]
```diff
@@ -71,7 +297,7 @@ <h4>Bernard Parah</h4>
                <h4>Carla Kirk-Cohen</h4>
                <a class="btn btn-dark btn-social mx-2" href="https://twitter.com/actuallyCarlaKC"><i class="fab fa-twitter"></i></a>
                <a class="btn btn-dark btn-social mx-2" href="https://github.com/carlakc"><i class="fab fa-github"></i></a>
-               <div class="bio-mentors">Carla builds infrastructure at <a target="_blank" href="https://lightning.engineering/">Lightning Labs</a> to make it easier to participate in a decentralized, private payments system.</div>
+               <div class="bio-mentors">Carla is a board member of <a target="_blank" href="https://btrust.tech/">₿Trust</a> and <a target="_blank" href="https://brink.dev/">Brink</a> and previously built infrastructure at <a target="_blank" href="https://lightning.engineering/">Lightning Labs</a>.</div>
```
Comment (Zero-1729) [inline index.html]:
```suggestion
               <div class="bio-mentors">Carla is a board member of <a target="_blank" href="https://btrust.tech/">₿trust</a> and <a target="_blank" href="https://brink.dev/">Brink</a> and previously built infrastructure at <a target="_blank" href="https://lightning.engineering/">Lightning Labs</a>.</div>
```

File: index.html [hunk 2]
```diff
@@ -89,7 +315,7 @@ <h4>Tim Akinbo</h4>
                <h4>Abubakar Nur Khalil</h4>
                <a class="btn btn-dark btn-social mx-2" href="https://twitter.com/ihate1999"><i class="fab fa-twitter"></i></a>
                <a class="btn btn-dark btn-social mx-2" href="https://github.com/Zero-1729"><i class="fab fa-github"></i></a>
-               <div class="bio-mentors">Abubakar is the CEO of <a target="_blank" href="https://rcrsv.xyz">Recursive Capital</a> and a Bitcoin Core contributor.</div>
+               <div class="bio-mentors">Abubakar is the CEO of <a target="_blank" href="https://rcrsv.xyz">Recursive Capital</a>, a board member of <a target="_blank" href="https://btrust.tech/">₿Trust</a>, and a Bitcoin Core contributor.</div>
```
Comment (Zero-1729) [inline index.html]:
```suggestion
               <div class="bio-mentors">Abubakar is the CEO of <a target="_blank" href="https://rcrsv.xyz">Recursive Capital</a>, a board member of <a target="_blank" href="https://btrust.tech/">₿trust</a>, and a Bitcoin Core contributor.</div>
```

**Review by Zero-1729 — COMMENTED** [review]
LGTM.

**nit**: Added minor suggestion on "`₿Trust`" to lowercase "`t`" just for [consistency](https://btrust.tech).

**Review by carlaKC — APPROVED** [review]

**Pull request merged**